src/main/webapp/pages/samples/sample_files.html

```diff
@@ -182,6 +182,58 @@ <h4 class="media-heading" th:text="#{samples.files.empty.title}"></h4>
 			</table>
 		</div>
 	</div>
+
+	<div class="row" th:if="${not #lists.isEmpty(fast5)}">
+		<div class="col-md-12">
+			<h2 th:text="#{sample.files.fast5}">FAST5 Files</h2>
+			<table class="table fast5" th:with="dateFormat=#{locale.date.short}, url=@{${#httpServletRequest.requestURI}}">
```

Comment:
do we need this `url` variable for anything?  doesn't seem to be used anywhere

Reply:
Updated in fbbf5f28d102ec8438c3e2ce9477e8ff2a9be5cf